### PR TITLE
Document existing security mechanisms

### PR DESCRIPTION
**Pull request by yurishkuro**
The situation is not ideal, but it's better to be transparent and ask for help.

### CODE REVIEW

**Comment by yurishkuro** [comment]
cc @objectiser

File: content/docs/1.18/security.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+* [ ] Sending trace data over UDP - no TLS/authentication.
+* [ ] Retrieving sampling configuration via HTTP - no TLS/authentication.
+
+## Client to Collector
+
+Clients can be configured to communicate directly with the Collector via HTTP. Unfortunately, at this time the Jaeger backend does not provide means of configuring TLS for its HTTP servers. The connections can be secured by using a reverse proxy placed in front of the collectors.
+
+* [ ] HTTP - no TLS/authentication.
+  * Some Jaeger clients support passing auth-tokens or basic auth.
```
Comment (pavolloffay) [inline content/docs/1.18/security.md:20]: We could add a link to client features page.

File: content/docs/1.18/security.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  * Blog post: [Secure architecture for Jaeger with Apache httpd reverse proxy on OpenShift](https://medium.com/@larsmilland01/secure-architecture-for-jaeger-with-apache-httpd-reverse-proxy-on-openshift-f31983fad400).
+
+## Agent to Collector
+
+* [x] gRPC - TLS with client cert authentication supported.
+* Blog post: [Protecting the collection of spans (using Keycloak)](https://medium.com/jaegertracing/protecting-the-collection-of-spans-1948d88682e5).
```
Comment (pavolloffay) [inline content/docs/1.18/security.md:26]: This blog post should be linked to the previous section https://medium.com/jaegertracing/protecting-the-collection-of-spans-1948d88682e5.

File: content/docs/1.18/security.md (hunk 1 above)
Comment (yurishkuro) [reply content/docs/1.18/security.md]: added the link, but the anchor doesn't work great with a table with sticky header

File: content/docs/1.18/security.md (hunk 2 above)
Comment (yurishkuro) [reply content/docs/1.18/security.md]: moved

**Pull request merged**